docs/section4/4-11.md

```diff
@@ -0,0 +1,65 @@
+            - Get dictionary of zones and hvac systems associated with the floor name: `dict_of_zones_hvac_systems_serving_specific_floor = get_dict_of_zones_hvac_sys_serving_specific_floor(floor_name,B_RMR)`
+            - Create list of hvac system serving the floor from the dictionary: `list_hvac_sys_serving_floor = list(dict_of_zones_hvac_systems_serving_specific_floor.values())`
+            - Loop through and check if any systems are of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `for hvac_flr in list_hvac_sys_serving_floor:`
+                - Check if the system is of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `if baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_flr.id)] in ["SYS-5","SYS-5b","SYS-6, SYS-6b", SYS-7",SYS-7a",SYS-7b",SYS-7c",SYS-8",SYS-8a",SYS-8b",SYS-8c"]:`
+                    - Set hvac_type_check boolean variable to TRUE: `hvac_type_check = TRUE`  
+            - Check if there is system types 5,6,7,8 on the floor via the hvac_type_check boolean variable, if true then check if the proposed zone is served by a multizone HVAC system: `if hvac_type_check == TRUE:`
+                - Check if the zone is served by a multizone system in the proposed by first getting the list of hvac systems serving the zone in the proposed: `list_hvac_systems_p = get_list_hvac_systems_associated_with_zone(P_RMR, zone_p.id)`
+                - Loop through each hvac system and check if any serve multiple zones: `for hvac_pr in list_hvac_systems_p:`
+                    - Check if the hvac system serves more than one zone: `if len(list(dict_hvac_sys_zones_served_p[hvac_pr.id]["Zone_List"].values())) > 1:`
+                        - Loop through terminal units associated with the zone to determine which is/are served by this HVAC system and check if it has reheat: `for terminal_p in zone_p.terminals:`
+                            - Check if the terminal unit is served by the HVAC system: `if terminal_p.served_by_heating_ventilation_air_conditioning_system == hvac_pr.id:`
+                                - Check if it has reheat, if it does then assume its a multizone unit and set the proposed served by multizone boolean variable to TRUE: `if terminal_p.heating_capacity >0:proposed_served_by_multizone = TRUE`   
+                                
+    **Rule Assertion:**
+    - Case 1: For each applicable zone, if no schedule mismatch is observed then pass: `if schedule_mismatch == FALSE: PASS`
+    - Case 2: Else if, there is a fan schedule mismatch but Section G3.1.1(c) appears to apply then Pass: `Elif schedule_mismatch == TRUE AND hvac_type_check == TRUE AND proposed_served_by_multizone == TRUE: PASS and raise_message "THERE IS A SCHEDULE MISMATCH BETWEEN THE BASELINE AND PROPOSED BUT SECTION G3.1.1(c) APPEARS APPLICABLE. VERIFY THAT THE FAN OPERATING SCHEDULE IN THE BASELINE IS IN ALIGNMENT WITH THE OCCUPANCY SCHEDULES."`
```

Comment:
Also, what if the single zone is being modeled due to different internal loads rather than different schedules? Then there may not be a schedule mismatch.